### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sebgoa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,39 @@
...
+```
+$ kubectl get pods -n transformation | grep transformation
+transformation-controller-6bdc658bf8-pwblp                1/1     Running   0          5d19h
+```
+
+A custom resource of kind `Transformation` can now be created, check a [sample](https://github.com/triggermesh/bumblebee/blob/master/config/samples/simple-event/transformation.yaml).
 
 ## Operations
 
+Current Bumblebee supports the following basic transformation operations:
```
Comment (sebgoa) [inline README.md:35]:
```suggestion
Currently Bumblebee supports the following basic transformation operations:
```

File: README.md [hunk 2]
```diff
@@ -1,15 +1,39 @@
-# CloudEvents Transformation
+# Bumblebee: A CloudEvents Transformer
 
-Transformation is an addressable CR based on Knative Serving 
-aimed on flexible CloudEvents modifications. When you create 
-Transformation object controller creates Knative Service that 
-accepts CloudEvents, applies transformation and replies with 
+Bumblebee allows you to transform [CloudEvents](https://cloudevents.io/) with an object of kind `Transformation`. A Transformation is an addressable custom resource based on Knative Serving
```
Comment (sebgoa) [inline README.md:3]:
```suggestion
Bumblebee allows you to transform [CloudEvents](https://cloudevents.io/) via an object of kind `Transformation`. A Transformation object creates an addressable custom resource based on Knative Serving 
```

File: README.md [hunk 3]
```diff
@@ -1,15 +1,39 @@
...
+# Bumblebee: A CloudEvents Transformer
 
-Transformation is an addressable CR based on Knative Serving 
-aimed on flexible CloudEvents modifications. When you create 
-Transformation object controller creates Knative Service that 
-accepts CloudEvents, applies transformation and replies with 
+Bumblebee allows you to transform [CloudEvents](https://cloudevents.io/) via an object of kind `Transformation`. A Transformation object creates an addressable custom resource based on Knative Serving 
+aimed at flexible CloudEvents modifications. When you create a
+Transformation object, the controller creates a Knative Service that 
+accepts CloudEvents, applies the declared transformation and replies with
```
Comment (antoineco) [inline README.md:6]:
```suggestion
accepts CloudEvents, applies the declared transformation and replies with a
```

File: README.md [hunk 4]
```diff
@@ -1,15 +1,39 @@
...
+Bumblebee can be compiled and deployed from source with [ko](https://github.com/google/ko):
+
+```
+ko apply -f ./config
+```
+
+You can verify that it installed successfully by checking the CRD:
+
+```
+$ kubectl get crd | grep transformation
```
Comment (antoineco) [inline README.md:20]:
```suggestion
$ kubectl get crd transformations.flow.triggermesh.io
```

File: README.md [hunk 5]
```diff
@@ -1,15 +1,39 @@
...
+transformations.flow.triggermesh.io                  2020-08-19T13:13:09Z
+```
+
+And checking that the controller is running:
+
+```
+$ kubectl get pods -n transformation | grep transformation
```
Comment (antoineco) [inline README.md:27]:
```suggestion
$ kubectl get pods -n transformation -l app=transformation-controller
```

File: README.md [hunk 6]
```diff
@@ -207,81 +231,14 @@ spec:
       value: $ceType
 ```
 
-## Events routing
+## Sample with Event Routing
 
-The CE with JSON payload being routed to Transformation CR where
-it gets modified according to the Specs and then being routed back
-to sender:
+Transformations are useful to modify the payload and cloudevent event attributes when an event is routed to a Target (aka event sink) that needs to receive a specific event type and payload. The cloudevent can be routed to a Transformation addressable via a specific Trigger where
```
Comment (antoineco) [inline README.md:236]:
```suggestion
Transformations are useful to modify the payload and CloudEvent context attributes when an event is routed to a Target (aka event sink) that needs to receive a specific event type and payload. The CloudEvent can be routed to a Transformation addressable via a specific Trigger where
```

**Review by antoineco — COMMENTED** [review]
A few suggestions.

**Review by antoineco — APPROVED** [review]

**Comment by tzununbekov** [comment]
If we don't want to show the readme part with our close source target, maybe it worth removing manifests from the sample `config/samples/multi-target` ?

**Pull request merged**